src/locales/en/common.json

```diff
@@ -91,7 +91,7 @@
   "Connected to:": "Connected to:",
   "Continue": "Continue",
   "Continue to Add account": "Continue to Add account",
-  "Continue to Dashboard": "Continue to Dashboard",
+  "Continue to Wallet": "Continue to Wallet",
```

Comment:
'Continue to Dashboard' is a default props, why its removed?

Reply:
It's not. common.js is regenerated every time the app runs and updated to whatever the current value is.